### ISSUE TITLE
InternalCompilerError: passing a dynamic calldata array to the constructor

### ISSUE THREAD

**Issue by iAmMichaelConnor** [ethereum/solidity#9663]
## Description

A constructor which takes a dynamic array as input fails to compile, with an `InternalCompilerError`.

## Environment

- Compiler version: solc version 0.6.10 (via truffle)
- Target EVM version (as per compiler settings): solc version 0.6.10
- Framework/IDE (e.g. Truffle or Remix): Truffle
- EVM execution environment / backend / blockchain client: truffle's default backend
- Operating system: macOS Catalina 10.15.6

## Steps to Reproduce

```solidity
pragma solidity ^0.6.10;

contract TestConstructor {

  constructor ( uint256[] calldata testInputs ) public {}

}
```

When running `truffle compile`:
<img width="404" alt="image" src="https://user-images.githubusercontent.com/42943676/91039408-6bbc9480-e604-11ea-8525-2147ef5e7de5.png">

**Comment by chriseth** [comment]
It should not be possible to use `calldata` for constructor parameters.

**Comment by a3d4** [comment]
I believe it was fixed in **6.11**, see PR #9319.
